public/assets/javascript/check_in.js

```diff
@@ -129,28 +129,27 @@ var checkinApp = new Vue({
 
             navigator.mediaDevices.getUserMedia({
                 video: {
-                    facingMode: 'environment'
+                    facingMode: "environment" //TODO: Add switch to toggle "user" or "enviroment"
```

Comment:
Please remove the `TODO` in favour of creating an issue with a more detailed description?